src/fidasim.f90

```diff
@@ -725,7 +779,7 @@ module libfida
         !+ Number of particles that hit a detector
     integer(Int32) :: nmax = 1000000
         !+ Maximum allowed number of particles grows if necessary
-    integer(Int32) :: nenergy = 100
+    integer(Int32) :: nenergy = 122
```

Comment:
`122`? How about `125` or better yet instead of changing the number of elements just change the max energy for the MC-NPA energy array allocation.